seedlet-blocks/sass/theme.scss

```diff
@@ -1,6 +1,14 @@
 @import './blocks/site-title.scss';
 @import './blocks/links.scss';
 
+/**
+ * Paragraph Margins
+ */
+
+p {
+	margin-top: 0;
```

Comment:
This is pretty generic. Are we sure it's what we want?

For example, if I place a cover block followed by a paragraph: 

<img width="683" alt="Screen Shot 2021-03-31 at 11 26 20 AM" src="https://user-images.githubusercontent.com/5375500/113169969-f7ca1680-91fa-11eb-864c-8b26610362b7.png">